src/port/port.tcl

```diff
@@ -2440,7 +2440,7 @@ proc action_notes { action portlist opts } {
                 puts [wrap $note 0 "  " 1]
             }
         } else {
-            puts "$portname has no notes."
+            ui_info "$portname has no notes."
```

Comment:
Ideally we'd hide this if run on a list of ports, but would still display this when run on a single port.

With this change, the current

```
$> port notes aquaterm
aquaterm has no notes.
```

would now become

```
$> port notes aquaterm
$>
```

which doesn't strike me as very user friendly :/

Reply:
Yeah, I agree; I adjusted it so that the notes command simply adds the `UI_PREFIX` to the status message; this is the simplest solution I could think of without having to treat pseudoports or names specially.